### DISCUSSION TITLE
How to train with custom features?

### DISCUSSION THREAD

**Discussion by theCodinCowboy** [explosion/spaCy#9328] [Help: Coding & Implementations] [answered]
How can I augment your classification and NER models with my own custom features? I have features I would like to append to the features currently used in the models. Is there a way to do this?

**Comment by polm** [comment, marked as answer]
There is not a straightforward way to do this at the moment, though [some FAQ entries](https://github.com/explosion/spaCy/discussions/8226#:~:text=i%20want%20to%20add%20non-textual%20features) describe how it can be done. It should get a little easier with #9069, which will be in 3.2.
